tests/functional/send_sms.js

```diff
@@ -134,14 +137,35 @@
      const formattedPhoneNumber =
        countryInfo.format(countryInfo.normalize(testPhoneNumber));
 
-     suite['valid phone number'] = function () {
+     suite['valid phone number, back'] = function () {
        return this.remote
         .then(openPage(SEND_SMS_URL, SELECTOR_SEND_SMS_HEADER))
         .then(type(SELECTOR_SEND_SMS_PHONE_NUMBER, testPhoneNumber))
         .then(click(SELECTOR_SEND_SMS_SUBMIT))
         .then(testElementExists(SELECTOR_SMS_SENT_HEADER))
         .then(testElementTextInclude(SELECTOR_SMS_SENT_TO, formattedPhoneNumber))
-        .then(testElementExists(SELECTOR_MARKETING_LINK));
+        .then(testElementExists(SELECTOR_MARKETING_LINK))
+
+        // user realizes they made a mistake
+        .then(click(SELECTOR_SMS_SENT_BACK))
+        .then(testElementExists(SELECTOR_SEND_SMS_HEADER))
+
+        // original phone number should still be in place
+        .then(testElementValueEquals(SELECTOR_SEND_SMS_PHONE_NUMBER, testPhoneNumber));
+     };
+
+     suite['valid phone number, resend'] = function () {
+       return this.remote
+        .then(openPage(SEND_SMS_URL, SELECTOR_SEND_SMS_HEADER))
+        .then(type(SELECTOR_SEND_SMS_PHONE_NUMBER, testPhoneNumber))
+        .then(click(SELECTOR_SEND_SMS_SUBMIT))
+        .then(testElementExists(SELECTOR_SMS_SENT_HEADER))
+
+        // Give a slight delay or else nexmo throttles the request
+        .sleep(10000)
```

Comment:
That sounds good. Doesn't have to go in this PR, just my spider sense going off for when I saw 10seconds.